doc/user_guide/encryption.rst

```diff
@@ -46,6 +49,12 @@ Specific examples
                        , user='user'
                        , password='password')
 
+   .. code-block:: python
```

Comment:
I think, we should mention, that this is for SaaS

Reply:
:heavy_check_mark: Moved it into SaaS example section